### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by YHDiamond**
there were a few code mistakes in this previous contributing.md which probably made a bad impression. so I fixed that and added a few more things

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,29 +11,36 @@ Thank for for considering a contribution! Generally, Geyser welcomes PRs from ev
...
+    private static final int AIR_ITEM = 0; // Static item names should be capitalized
+
+    public Int2IntMap items = new Int2IntOpenHashMap(); // Use the interface as the class type but initialize with the implementation.
+
+    public int nameWithMultipleWords = 0;
+
+    /**
+    * Javadoc comment to explain what a function does.
+    */
+    @RandomAnnotation(AnnotationStuff)
```
Comment (theminecoder) [inline CONTRIBUTING.md:24]: This won't compile ;)

```suggestion
    @RandomAnnotation(stuff = true, moreStuff = "might exist")
```

**Review by theminecoder — CHANGES_REQUESTED** [review]

**Comment by YHDiamond** [comment]
Any news on this

**Comment by YHDiamond** [comment]
Are there any compilation issues (besides using variable names that don't exist, etc) that I should fix?

**Comment by YHDiamond** [comment]
Just to note: @lukeeey sent a form for contributors to fill out, and one of the questions asked about the CONTRIBUTING.md. This will hopefully make people's 'review' of the CONTRIBUTING.md 10/10

**Comment by lukeeey** [comment]
@YHDiamond I actually discovered the other day that the contributing guidelines aren't just for how you format your code. They can also cover creating PRs as well as issues, but don't worry about that right now.

Here is some very good contribution guidelines: https://github.com/PaperMC/Paper/blob/master/CONTRIBUTING.md

**Comment by YHDiamond** [comment]
Ok, but the contributing guidelines should contain good code, right? If they don't contain good code it is embarrassing (imo).

**Comment by lukeeey** [comment]
Yeah

**Review by lukeeey — APPROVED** [review]

**Comment by YHDiamond** [comment]
thank you.  now we must wait for the merge power

**Comment by YHDiamond** [comment]
Any news on this? Will it be merged? 
image lol though

**Pull request merged**